app/bundles/AssetBundle/Event/AsssetExportListEvent.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\AssetBundle\Event;
+
+use Mautic\CoreBundle\Event\CommonEvent;
+
+final class AsssetExportListEvent extends CommonEvent
```

Comment:
```suggestion
final class AssetExportListEvent extends CommonEvent
```
Looks like a small typo, should be two s letters, not three. Also, don’t forget to rename the file accordingly.

Reply:
fixed